docs/user-guide/exploring-stories.md

```diff
@@ -13,7 +13,7 @@ The user can approach a story in two different ways:
 ## Edit Mode
 
 The *Edit Mode* allows the user to edit a story by adding, removing or modifying the elements inside it. This mode and its tools are used both to edit an existing story and to create a new one. <br>
-In order to create a new story, the user can click on the **New GeoStory** button <img src="../img/button/new_story_button.jpg" class="ms-docbutton"/> on [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) home page. As soon as the user clicks on that button in home page the story editor opens, it is composed of two main elements: the *Builder* and the *Sections Container* (later simply called *Container*).
+In order to create a new story, the user can click on the **New GeoStory** button <img src="../img/button/new_story_button.jpg" class="ms-docbutton"/> on [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) home page. As soon as the user clicks on that button in home page the story editor opens, it is composed of three main elements: the [Topbar](exploring-dashboards.md#topbar), the *Builder* and the *Sections Container* (later simply called *Container*).
```

Comment:
I think it is not a good idea to link to the topbar of dashboard since it contains some specific plugins not included in stories like **Delete Dashboard**, I would create a dedicate list here too because in the future they may differ
![image](https://user-images.githubusercontent.com/11991428/191510600-dd22172e-e5b2-49d5-b578-da8a7a068ddf.png)